pkg/storegateway/indexheader/fileutil/mmap_windows.go

```diff
@@ -0,0 +1,38 @@
+// SPDX-License-Identifier: AGPL-3.0-only
+// Provenance-includes-location: https://github.com/prometheus/prometheus/blob/main/tsdb/fileutil/mmap_windows.go
+// Provenance-includes-license: Apache-2.0
+// Provenance-includes-copyright: The Prometheus Authors.
+
+package fileutil
+
+import (
+	"os"
+	"syscall"
+	"unsafe"
+)
+
+func mmap(f *os.File, size int) ([]byte, error) {
```

Comment:
Even though we don't support Windows, should we add a `bool` param here for symmetry and in anticipation of potentially upstreaming this?

Reply:
if we don't support windows, do we need that file at all?

Reply:
Good point, I guess not!

Reply:
I've deleted it :) It's Friday...